packages/@aws-cdk/aws-s3/lib/bucket.ts

```diff
@@ -827,6 +842,13 @@ export interface BucketProps {
    */
   readonly lifecycleRules?: LifecycleRule[];
 
+  /**
+   * Whether this bucket should have server access logs configured or not.
+   *
+   * @default - Disabled server access logs
+   */
+  readonly serverAccessLogs?: ServerAccessLogs;
+
```

Comment:
We try to avoid nesting if possible to make the common case less verbose, so let's get rid of the `ServerAccessLogs` struct and just add two properties under `BucketProps`:

```ts
interface BucketProps {
  // ...
  readonly serverAccessLogsBucket?: s3.IBucket;
  readonly serverAccessLogsPrefix?: string;
}
```